src/views/ScorePage/ScorePage.js

```diff
@@ -8,31 +8,52 @@ import Answer from '../../components/Answer/Answer';
 
 export function renderScorePage() {
   const recentUserScore = getCurrentScore(userAnswers);
-
+  const totalNumber = QuizSettings.questionsNum;
   const container = document.createElement('div');
   container.setAttribute('id', 'scorePage');
   container.append(
     renderGraphics(),
-    renderScore(recentUserScore),
+    renderScore(recentUserScore, totalNumber),
     renderNickForm(),
     renderMenuButton(),
     renderPlayAgainButton(),
   );
   document.querySelector('#app').append(container);
+  renderCongratsMessage(recentUserScore, totalNumber);
 }
 
-function renderScore(score) {
+function renderScore(score, totalNumber) {
   const container = document.createElement('div');
   const congratsText = document.createElement('h2');
   const scoreText = document.createElement('h2');
   const finalScore = document.createElement('h2');
   finalScore.setAttribute('id', 'finalScore');
-  congratsText.innerText = 'CONGRATS!';
-  scoreText.innerText = 'YOUR SCORE IS: ' + score;
+  congratsText.setAttribute('id', 'congratsText');
+  congratsText.innerText = '';
+  scoreText.innerText = `YOUR SCORE IS:  ${score} / ${totalNumber}`;
   container.append(congratsText, scoreText, finalScore);
   return container;
 }
 
+function renderCongratsMessage(recentUserScore, totalNumber) {
+  let message;
+  const finalMessage = document.getElementById('congratsText');
+  finalMessage.innerText = getCongratsMessage(message);
+
+  function getCongratsMessage() {
+    const totalPercent = Math.floor((recentUserScore / totalNumber) * 100);
+    if (totalPercent >= 80) {
+      message = '"WOW"! Your score is ' + totalPercent + '%! You are True animal lover!';
+    } else if (totalPercent >= 50) {
+      message =
+        'You could use a litte practice! Your score is ' + totalPercent + '% Would you like to do the quiz again?';
+    } else {
+      message = 'Well, it could be better! Your score is ' + totalPercent + '% Would you like to do the quiz again?';
+    }
+    return message;
+  }
+}
+
```

Comment:
The function inside a function is still unreadable. Additionally, getCongratsMesage returns a message so there is no use in let message; in the "parent" function. 

Please move getCongratsMessage outside renderCongratsMessage block and remove let message from renderCongratsMessage